### DISCUSSION TITLE
MC -> Paste Migration

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2632] [General] [answered]
Hello! I’m in the process of organizing MC’s migration to Paste, and am looking for guidance on components/workflows that don’t exist yet in Paste, but are frequently used in SendGrid. I’ve been combing through the [[Gap Analysis doc](https://docs.google.com/document/d/1LUw-bonxohCFuL2BoT5om47tFbLxwhNOLgXXyDwjQ64/edit#)] and am wondering if I should attend office hours to more formally kick this project off, get Paste’s perspective on my general approach, and gain more context on the doc. 

I have questions surrounding specific components/workflows that I’m happy to create separate tickets for as well, but thought this might be a good place to start - let me know what you think. Thanks!

**Comment by shleewhite** [comment, marked as answer]
Hey @ceyerkes!

Office hours would be a great way to talk about this. For the gap analysis, we recently released [Toggle Button](https://paste.twilio.design/components/button#toggle-button) and are currently working on Switch.

**Reply by ceyerkes** [reply]
Sounds good, I'll sign up!
